### DISCUSSION TITLE
Factory Sample Slots

### DISCUSSION THREAD

**Discussion by Maxklui** [bangcorrupt/hacktribe#169] [Q&A] [answered]
@bangcorrupt 

Hello

 I have another question about the hacktribe on the Electribe sampler: will the factory sample slots be retained? Because the header is not modified there. As far as I have read. 

Many thanks for the help

**Comment by Piergap** [comment, marked as answer]
the new waveforms will be placed in the slots from 1 to 274, then the original presets in those slots need to be moved in the higher positions.
It is reccomended to move the presents to the higher slots (which then affects the patterns programmed with the standard settings, though).
You can find the explanation of the process in the tutorial here at min 2:40
https://youtu.be/Ep0c7jM9d3M
